### PR TITLE
Update customer-success.md

### PR DESCRIPTION
**Pull request by MichaelBDavis**
Update churn playbook

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [x] Suitable unit/system level tests have been added and they pass
 - [x] Documentation has been updated

### CODE REVIEW

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

File: src/handbook/customer/customer-success.md [hunk 1]
```diff
@@ -178,9 +178,24 @@ and getting more value from the product
...
 fixes.
 
-## Churned customers
+## Procedure for Handling a Churned Customer
 
-When an annually-billed customer chooses not to renew their subscription, the assigned Account Executive should login into Stripe and cancel the subscription. This will effectively close the account on the platform and prevent invoices to continue being sent.
+The Assigned Account Executive is responsible for executing the following steps when an annually-billed customer chooses not to renew their subscription:
+
+1. **Update Contracted Customer Tracker**
+   Access the Contracted Customer Tracker and move the churned customer to the "Churned" tab.
```
[the remark this answers is not in the document]
Comment (MichaelBDavis) [reply src/handbook/customer/customer-success.md:186]: Yes, do i click "start a review" to make this change?

[the remark this answers is not in the document]
Comment (ZJvandeWeg) [reply src/handbook/customer/customer-success.md:186]: Screenshot 2024-10-17 at 10 29 54
If you click that and update the text to what it should do, you can coment that and accept your own suggestion. Links in markdown are written as: `[Link text](https://your.location.tld/)`

[the remark this answers is not in the document]
Comment (MichaelBDavis) [reply src/handbook/customer/customer-success.md:186]: Access the [Contracted Customer Tracker](https://docs.google.com/spreadsheets/d/1QcGVeiO_RXwYHCZvGcuBW3oYgR5NMsQ22yi3cOFY73o/edit?gid=1852075496#gid=1852075496) and move the churned customer to the "Churned" tab.

[the remark this answers is not in the document]
Comment (ZJvandeWeg) [reply src/handbook/customer/customer-success.md:186]:
```suggestion
   Access the [Contracted Customer Tracker](https://docs.google.com/spreadsheets/d/1QcGVeiO_RXwYHCZvGcuBW3oYgR5NMsQ22yi3cOFY73o/edit?gid=1852075496#gid=1852075496) and move the churned customer to the "Churned" tab.
```

**Review by ZJvandeWeg — APPROVED** [review]

**Pull request merged**